### PR TITLE
add priv step1

### PR DESCRIPTION
**Pull request by shenli**
Prepare for grant statement.

### CODE REVIEW

File: util/types/etc.go [hunk 1]
```diff
@@ -147,6 +147,10 @@ func TypeToStr(tp byte, binary bool) string {
 		return "timestamp"
 	case mysql.TypeBit:
 		return "bit"
+	case mysql.TypeEnum:
```
Comment (siddontang) [inline util/types/etc.go]: here may be in another PR

**Comment by ngaut** [comment]
LGTM

File: util/types/etc.go (hunk 1 above)
Comment (shenli) [inline util/types/etc.go]: It's better. But I am blocked here and can not wait.

**Comment by siddontang** [comment]
LGTM
